SPECS/systemd/50-security-hardening.conf

```diff
@@ -17,3 +17,16 @@ net.ipv4.ip_forward = 0
 net.ipv6.conf.all.forwarding = 0
 #Disable IPv6 Router Advertisements
 net.ipv6.conf.all.accept_ra = 0
+# Security patch for CCE-4186-3, msid: 38.4
```

Comment:
This may be disruptive to current users, probably move this to the post image. (This stuff should probably happen in the switch to Mariner 3.0)